services/notifications/src/simcore_service_notifications/services/postgres/_health.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from asyncio import Task
+from datetime import timedelta
+
+from fastapi import FastAPI
+from models_library.healthchecks import IsResponsive, LivenessResult
+from servicelib.async_utils import cancel_wait_task
+from servicelib.background_task import create_periodic_task
+from servicelib.db_asyncpg_utils import check_postgres_liveness
+from servicelib.fastapi.db_asyncpg_engine import get_engine
+from servicelib.logging_utils import log_catch
+
+_logger = logging.getLogger(__name__)
+
+
+class PostgresHealth:
```

Comment:
Would be nice to have "liveness" in the name of this health check. We mix all the time these concepts . I also do not know how many implementations we have around of the same thing. I wonder if we should do a similar effort as with the lifespan and move them all to servicelib instead of copy&pasting them around